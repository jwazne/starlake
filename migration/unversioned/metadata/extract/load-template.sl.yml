load:
  database: "{{custom_project_id}}"
  name: "DOMAIN"
  rename: "raw_{{normalized_domain}}"
  metadata:
    encoding: "ISO-8859-1"
    mode: "FILE"
    format: "DSV"
    multiline: false
    array: false
    withHeader: true
    separator: ";"
    quote: "\""
    escape: "\""
    write: "OVERWRITE"
  tables:
    - name: "User"
      pattern: "table.*.csv"
      attributes:
        - name: "id"
          type: "long"
          array: false
          required: false
          privacy: "NONE"
          trim: "BOTH"
