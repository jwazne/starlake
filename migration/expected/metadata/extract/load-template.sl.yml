---
version: 1
load:
  database: "{{custom_project_id}}"
  name: "DOMAIN"
  rename: "raw_{{normalized_domain}}"
  metadata:
    encoding: "ISO-8859-1"
    format: "DSV"
    multiline: false
    array: false
    withHeader: true
    separator: ";"
    quote: "\""
    escape: "\""
    writeStrategy:
      type: "OVERWRITE"
